cli/storage_profile.c

```diff
@@ -257,7 +257,7 @@ perform_profile_run(
         work_specs[i].tmpfs = tmpfs;
         work_specs[i].rc = 0;
 
-        work_specs[i].samples = malloc(samples_per_thread * sizeof(double));
+        work_specs[i].samples = malloc(samples_per_thread * sizeof(*work_specs[i].samples));
```

Comment:
what was clang-tidy's complaint here?

Reply:
samples is a uint64_t array, not a double array, so it complained about sizeof(double)

Reply:
Righteous!